src/components/SignUp/SignUp.tsx

```diff
@@ -49,35 +51,43 @@ export function SignUp(): React.ReactElement {
     </div>
   );
 
+  let handleValue: string = '';
```

Comment:
Please do not do this. This is not how state is properly handled in React. Instead you should use the `useState` hook.